lab4-Electronics_Hardware_and_more/lab4-code/sensor_integration_example/sensor_integration_example.ino

```diff
@@ -0,0 +1,24 @@
+// Title:         Sensor Inegration Example
+// Last Updated:  October 25th, 2021
+// Author:        Emma Paczkowski
+// Breif:         This program integrated all sensors used in the Space School Fall of 2021 program into a single sript as an example of what could be done with the sensor suite
+// set the analog input pin
+int input_pin = 0;
+
+
+void setup()
+{
+}
+
+void loop()
+{
+   // read the analog input
+   int analog_input = analogRead(input_pin);
+  
+   // convert analog input to a voltage value
+   int voltage_reading = analog_input * (5.0/1024.0); 
+  
+   // convert voltage value to a temperature value
+   int temperatureC = (voltage_reading - 0.5) * 100; 
+   Serial.println(temperatureC);
+}
```

Comment:
in lab 6, there are two examples of multiple sensors, as well as the SD card, working together. So I am not sure if this is required. 

As well, I think the purpose of lab 4 should be just an introduction to the hardware, making sure they can get them to work, seeing what each sensor can do, and making sure they have all the software libraries ready to go. In the proceeding labs is when they will integrate multiple sensors together.

Reply:
this file is really old and was not supposed to be uploaded oops.  It has been deleted